tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/bala/globalvar/PublicVariableBalaTest.java

```diff
@@ -68,4 +78,20 @@ public void testModulePublicVariableAccessNegative() {
         validateError(compileResultNegetive, index++, "undefined symbol 'number'", 20, 20);
         assertEquals(compileResultNegetive.getErrorCount(), index);
     }
+
+    @Test(description = "Test the position of global variable available in a module.")
+    public void testModulePublicVariablePos() {
+        CompileResult birTestResult = BCompileUtil.compile(
```

Comment:
This is just a normal compilation result, so `birTestResult` is kinda misleading?

Reply:
Changed